src/ray/common/ray_config_def.h

```diff
@@ -102,6 +102,7 @@ RAY_CONFIG(uint64_t, task_oom_retries, -1)
 /// The worker killing policy to use, available options are
 /// group_by_owner
 /// retriable_lifo
+/// retriable_fifo
 RAY_CONFIG(std::string, worker_killing_policy, "group_by_owner")
```

Comment:
We can set this to retriable_fifo here
Or ask them to add this env var to their cluster config RAY_worker_killing_policy="retriable_fifo"

Reply:
ok get it. will try to set this in cluster env for now